cli_tools/gce_vm_image_import/importer/importer.go

```diff
@@ -162,24 +174,28 @@ func (i *importer) getCtxError(ctx context.Context) (err error) {
 }
 
 func (i *importer) cleanupDisk() {
-	if i.pd.uri == "" {
+	cleanupDisk(i.diskClient, i.project, i.zone, i.pd)
+}
+
+func cleanupDisk(diskClient diskClient, project string, zone string, pd persistentDisk) {
+	if pd.uri == "" {
 		return
 	}
 
-	diskName := path.Base(i.pd.uri)
-
-	if err := i.diskClient.DeleteDisk(i.project, i.zone, diskName); err != nil {
+	diskName := path.Base(pd.uri)
+	if err := diskClient.DeleteDisk(project, zone, diskName); err != nil {
 		gAPIErr, isGAPIErr := err.(*googleapi.Error)
 		if isGAPIErr && gAPIErr.Code != 404 {
-			log.Printf("Failed to remove temporary disk %v: %e", i.pd, err)
+			log.Printf("Failed to remove temporary disk %v: %e", pd, err)
 		}
 	}
 }
 
 func (i *importer) buildLoggable() service.Loggable {
 	return service.SingleImageImportLoggable(i.pd.sourceType, i.pd.sourceGb, i.pd.sizeGb,
 		i.pd.matchResult, i.pd.inflationType, int64(i.pd.inflationTime.Milliseconds()),
-		int64(i.pd.shadowInflationTime.Milliseconds()), i.traceLogs)
+		int64(i.pd.shadowInflationTime.Milliseconds()), i.pd.isUEFICompatible, i.pd.isUEFIDetected,
+		i.traceLogs)
```

Comment:
About `pd.isUEFI`:

I think you have the two different vars to support this metric. If so, then I think it's time to decouple the logging of key/value pairs. Right now, steps can expose traceLogs, and they're collected after running. You'd add another method to expose a map of key/value pairs, collecting them after each step runs.

Two big reasons:
1. SingleImageImportLoggable's signature is getting large, which leads to inadvertently swapped parameters.
2. Exposing implementation details on domain objects just for logging.

Pragmatically, if it were me, I'd do this to keep it simple:
1. Keep `SingleImageImportLoggable` with the high-level variables such as format, size, timing, etc. 
2. Add a method to insert additional key/value pairs
3. within `buildLoggable`, iterate over the key/value pairs that you collected from each step, and insert them.